### PR TITLE
changed minSdkVersion to 22 with linked #17 issue

### LINKED ISSUE #17: minSdkVersion is to high
In build gradle (module app) minSdkVersion is 26. That means that only users with android 8.0 will be able to install app. I suggest minSdkVersion 22: people with android 5.1 and above will be able to install app.
I can implement this

### PR DESCRIPTION
**Pull request by sakshi2707**
closes #17

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -2,13 +2,13 @@ apply plugin: 'com.android.application'
...
-    buildToolsVersion "30.0.1"
+    compileSdkVersion 23
+    buildToolsVersion "23.0.3"
 
     defaultConfig {
         applicationId "com.cswala.cswala"
-        minSdkVersion 26
-        targetSdkVersion 30
+        minSdkVersion 19
+        targetSdkVersion 23
```
Comment (riturajjain2000) [inline app/build.gradle:11]: no need of changing this

File: app/build.gradle [hunk 2]
```diff
@@ -2,13 +2,13 @@ apply plugin: 'com.android.application'
...
-    compileSdkVersion 30
-    buildToolsVersion "30.0.1"
+    compileSdkVersion 23
+    buildToolsVersion "23.0.3"
 
     defaultConfig {
         applicationId "com.cswala.cswala"
-        minSdkVersion 26
-        targetSdkVersion 30
+        minSdkVersion 19
```
Comment (riturajjain2000) [inline app/build.gradle:10]: change from 26 to 22

File: app/build.gradle [hunk 3]
```diff
@@ -2,13 +2,13 @@ apply plugin: 'com.android.application'
 apply plugin: 'com.google.gms.google-services'
 
 android {
-    compileSdkVersion 30
-    buildToolsVersion "30.0.1"
```
Comment (riturajjain2000) [inline app/build.gradle:6]: no need of changing this

File: app/build.gradle [hunk 4]
```diff
@@ -2,13 +2,13 @@ apply plugin: 'com.android.application'
 apply plugin: 'com.google.gms.google-services'
 
 android {
-    compileSdkVersion 30
```
Comment (riturajjain2000) [inline app/build.gradle:5]: no need of changing this

**Comment by sakshi2707** [comment]
closes #17

**Review by riturajjain2000 — APPROVED** [review]

**Pull request merged**